### PR TITLE
Tutorial Section Clean Up - Typo and Code Style Fix

### PR DESCRIPTION
**Pull request by Doppp**
Was going through the tutorial section and decided to just clean up some things. :)

### CODE REVIEW

File: documentation/04_tutorials/15-conclusion.html.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: "Conclusion"
 
 Now we have a pretty complete, working game! It's not too complex, but it should give you a full, well-rounded idea of how to make your own games in HaxeFlixel. I really hope you've enjoyed this tutorial, and take the time to play around with the code some more to try expanding the game even further. You could add more levels, add stairway objects to move between them, add more enemy types, and collectibles, and upgrades… the list goes on.
 
-You can grab the complete source code for the game [here](https://github.com/HaxeFlixel/flixel-demos/master/Tutorials/TurnBasedRPG/).
```
Comment (Gama11) [inline documentation/04_tutorials/15-conclusion.html.md]: Nice catch. Should link to the master, not the dev branch though.

File: documentation/04_tutorials/10-part-ix-ui-and-combat.html.md [hunk 2]
```diff
@@ -134,7 +134,7 @@ This all seems simple enough, but it's actually going to require several compone
 		FlxG.collide(_player, _mWalls);
 		FlxG.overlap(_player, _grpCoins, playerTouchCoin);
 		FlxG.collide(_grpEnemies, _mWalls);
-		checkEnemyVision();
+		checkEnemyVision;
```
Comment (Gama11) [inline documentation/04_tutorials/10-part-ix-ui-and-combat.html.md]: ?

File: documentation/01_community/03-introduction-to-haxe.html.md [hunk 3]
```diff
@@ -7,15 +7,15 @@ The Haxe programming language itself is High Level and Strictly Typed. It allows
...
 *   https://groups.google.com/forum/#!forum/haxelang
-*   IRC channel #haxe on freenode. 
+*   IRC channel #haxe on freenode.
 
 Haxe is being developed through GitHub:
 
 *   https://github.com/HaxeFoundation
 
-[![](/images/haxe-logo.png)](http://haxe.org)
+[![](/documentation/images/01_community/haxe-logo.png)](http://haxe.org)
```
Comment (Gama11) [inline documentation/01_community/03-introduction-to-haxe.html.md]: The other doc pages use `../images` - (which is necessary for it to work on the website, not just here on the repo).

File: documentation/04_tutorials/10-part-ix-ui-and-combat.html.md (hunk 2 above)
Comment (Doppp) [inline documentation/04_tutorials/10-part-ix-ui-and-combat.html.md]: I couldn't compile and run with the braces. Not sure if I am doing it right. I'm still new.

Comment (Gama11) [inline documentation/04_tutorials/10-part-ix-ui-and-combat.html.md]: It's not a function call without the braces, so it won't do anything.

http://try.haxe.org/#C15AC

What error were you getting?

Comment (Doppp) [inline documentation/04_tutorials/10-part-ix-ui-and-combat.html.md]: I got this:
```
source/PlayState.hx:109: characters 12-30 : Not enough arguments
Build halted with errors.
```

Comment (Gama11) [inline documentation/04_tutorials/10-part-ix-ui-and-combat.html.md]: Hm, that line is supposed to look like this:

https://github.com/HaxeFlixel/flixel-demos/blob/master/Tutorials/TurnBasedRPG/source/PlayState.hx#L145

Comment (Doppp) [inline documentation/04_tutorials/10-part-ix-ui-and-combat.html.md]: Alright, I'll use that. Thanks for catching it!

File: documentation/04_tutorials/15-conclusion.html.md (hunk 1 above)
Comment (Doppp) [inline documentation/04_tutorials/15-conclusion.html.md]: Thanks. Made the change.

**Comment by Gama11** [comment]
Thanks!

**Pull request merged**